### PR TITLE
Merge upstream v191

### LINKED ISSUE #14: [URGENT] Update with upstream from v168 to v191
Currently, deploying an Python application displays:

```
[LOG] !     Python has released a security update! Please consider upgrading to python-3.7.6
[LOG] Learn More: https://doc.scalingo.com/languages/python/start#supported-versions
```

Which is not nice.

URGENT to fix this security issue: https://www.cert.ssi.gouv.fr/avis/CERTFR-2021-AVI-140/

Notify when solved:
https://app.intercom.com/a/apps/w4oogu7s/inbox/inbox/all/conversations/12375700021612

### PR DESCRIPTION
**Pull request by EtienneM**
As compared to the upstream version, here are the differences:
- Keep the file `vendor/jq` as the stack scalingo-18:v14 is not yet released.
- Fix a bug in upstream buildpack in e95dcfc 
- Add support for BUILDPACK_DEBUG env variable in e193fb4 

Tested on the test-python-buildpack application. Deployed here (https://test-python-buildpack.osc-fr1.scalingo.io/), deployment logs there (https://dashboard-next.scalingo.com/apps/osc-fr1/test-python-buildpack/deploy/list):

```
 <-- Start deployment of test-python-buildpack -->
       Fetching source code
       Fetching deployment cache
-----> Cloning custom buildpack: https://github.com/Scalingo/python-buildpack#deps/14/upstream_v191
-----> Installing pip 20.1.1, setuptools 47.1.1 and wheel 0.34.2
       Skipping installation, as Pipfile.lock hasn't changed since last deploy.
-----> Installing SQLite3
-----> Procfile declares types -> web
 Build complete, shipping your container...
 Waiting for your application to boot... 
 <-- https://test-python-buildpack.osc-fr1.scalingo.io -->

```

Fix #14 / [OPSB-248]

[OPSB-248]: https://scalingo.atlassian.net/browse/OPSB-248

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -124,6 +94,33 @@ source "$BIN_DIR/utils"
 # shellcheck source=bin/warnings
 source "$BIN_DIR/warnings"
 
+if [[ "${STACK}" == "cedar-14" ]]; then
```
Comment (Soulou) [inline bin/compile:97]: Too much heroku-centric, should be adapted for Scalingo

File: bin/compile [hunk 2]
```diff
@@ -189,8 +191,6 @@ cp -R "$CACHE_DIR/.scalingo/python-stack" .scalingo/ &> /dev/null || true
 cp -R "$CACHE_DIR/.scalingo/python-version" .scalingo/ &> /dev/null || true
 # A plain text file which contains the current sqlite3 version being used (used for cache busting).
 cp -R "$CACHE_DIR/.scalingo/python-sqlite3-version" .scalingo/ &> /dev/null || true
-# Any pre-compiled binaries, provided by the buildpack.
-cp -R "$CACHE_DIR/.scalingo/vendor" .scalingo/ &> /dev/null || true
```
Comment (Soulou) [inline bin/compile:193]: By removing this we must be sure the new stack is deployed with `jq` in it.

**Review by Soulou — CHANGES_REQUESTED** [review]

File: bin/compile (hunk 2 above)
Comment (EtienneM) [reply bin/compile:193]: Nice catch 👍🏾

File: bin/compile [hunk 3]
```diff
@@ -124,6 +94,30 @@ source "$BIN_DIR/utils"
 # shellcheck source=bin/warnings
 source "$BIN_DIR/warnings"
 
+if [[ "${STACK}" == "cedar-14" ]] || [[ "${STACK}" == "scalingo-14" ]]; then
```
Comment (Soulou) [inline bin/compile:97]:
```suggestion
if [[ "${STACK}" == "cedar-14" || "${STACK}" == "scalingo-14" ]]; then
```

**Review by Soulou — CHANGES_REQUESTED** [review]

File: bin/compile (hunk 3 above)
Comment (EtienneM) [reply bin/compile:97]: Erf sorry 🤦🏾

**Review by Soulou — APPROVED** [review]

**Pull request merged**